tasks/windows.ps1

```diff
@@ -0,0 +1,46 @@
+[CmdletBinding()]
+param(
+  # NOTE: init.json cannot yet be shared, so must have windows.json / windows.ps1
+  [Parameter(Mandatory = $true)]
+  [String]
+  $Name,
+
+  [Parameter(Mandatory = $true)]
+  [ValidateSet('start', 'stop', 'restart', 'status')]
```

Comment:
You could avoid the tolower calls later on by using IgnoreCase: `[ValidateSet('start', 'stop', 'restart', 'status', IgnoreCase = $false)]`, which would require the case you specify. However this throws a `ParameterBindingValidationException` which kills execution before getting into the script if you use any variation of case other than what's in the set. This may be more user hostile than you intend.

Reply:
Just double checked - Puppet language definition of `Enum` is case-sensitive already per https://puppet.com/docs/puppet/5.3/lang_data_abstract.html#enum so setting `IgnoreCase = $false` would match how Bolt / PE tasks will be called by the system.

That said, that means it isn't super necessary one way or another either. (Even the `[ValidateSet]` is not protecting anything - just adding more code to keep in sync with `metadata.json`. I'll vet a couple scenarios locally where things are mismatched.)

As for the UX part, I believe the web interface uses the `Enum` values to fill a drop-down list, so that a user selects the value from a list rather than does raw data entry... at the CLI `bolt task show service::windows` should give similar info to callers, and validation should happen prior to calling the PS.